src/AwesomeAlert.js

```diff
@@ -56,12 +56,12 @@ export default class AwesomeAlert extends Component {
         toValue: 0,
         tension: 10,
         useNativeDriver,
-      }).start();
-
-      setTimeout(() => {
-        this._toggleAlert();
-        this._onDismiss();
-      }, 70);
+      }).start(({finished}) => {
+        if (finished) {
```

Comment:
@zkwentz Hey Zack, tested this change. Can you double-check the usage of the `finished` flag here, might have to remove its dependency. I'm looking at https://reactnative.dev/docs/animated#working-with-animations, focusing on 

> If the animation is done because stop() was called on it before it could finish (e.g. because it was interrupted by a gesture or another animation), then it will receive {finished: false}

IMO the alert should hide, regardless of the animation got done with/without interruption, please check. Let me know if it makes sense.